services/datamanager/datasync/upload_data_capture_file.go

```diff
@@ -18,6 +18,11 @@ func uploadDataCaptureFile(ctx context.Context, client v1.DataSyncServiceClient,
 		return err
 	}
 
+	// Do not attempt to upload a file without any sensor readings, which occurs when syncing more frequently than capturing.
```

Comment:
I think the post , part can be left out, since there are other cases where this would also apply (e.g. restarting/disabling/changing before first write)

Reply:
Updated!